Dockerfile

```diff
@@ -9,16 +9,33 @@ RUN apt update && apt install -y --no-install-recommends \
         openjdk-11-jdk-headless \
         wget \
         zip && \
-    rm -r /var/lib/apt/lists/* && \
-    rm -r /var/cache/apt/*
+    apt clean
+
+# install nextflow
+ADD https://get.nextflow.io nextflow
+RUN chmod +x nextflow && \
+    mv nextflow /usr/bin && \
+    nextflow self-update
+
+FROM base AS talos_gcloud
 
 # Google Cloud SDK: use the script-based installation, as the Debian package is outdated.
-RUN curl https://sdk.cloud.google.com > install.sh && \
-    bash install.sh --disable-prompts --install-dir=/opt && \
-    rm install.sh
+ADD https://sdk.cloud.google.com install_glcoud.sh
+RUN bash install_glcoud.sh --disable-prompts --install-dir=/opt && \
+    rm install_glcoud.sh
 
 ENV PATH=$PATH:/opt/google-cloud-sdk/bin
 
-COPY requirements*.txt README.md setup.py .
+# Add in the additional requirements that are most likely to change.
+COPY requirements*.txt README.md setup.py ./
+COPY src src/
+RUN pip install --upgrade pip && pip install .[cpg]
+
+FROM base AS talos_none
```

Comment:
Out of interest, why do you install twice  +/- gcloud?

Reply:
OR - I could just read the new bits in the README and learn for my self...

Reply:
Heh. This was something Miah wanted so he didn't have to keep customising the dockerfile to drop the 2GB Gcloud install. There's probably a sexier way to do this as a condition build, but polishing this file felt like a distraction